CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Linkage Checker Enforcer Rule Change Log
 
+## 1.3.0
+* LinkageCheckerMain has an option (`-o`) to output linkage errors into a file.
```

Comment:
How stable is this? I.e. how committed are we to the exact format and feature set? I'm wondering if this should be marketed as an alpha release. 

I was going to suggest we should add doc improvements in this PR, but maybe it would be better to have a separate PR ready to go with the doc changes before we release it. Writing the docs up front will also give us viewpoint on whether we have the design right for this feature.

Reply:
I will create a new Wiki page for LinkageCheckerMain class indicating this is alpha and will update this changelog in this PR.